lblod/frontend-gelinkt-notuleren
# Update to `@lblod/ember-rdfa-editor-lblod-plugin` 19.3.1 hotfix

### Overview
This (draft) PR contains an update to `@lblod/ember-rdfa-editor-lblod-plugin` 19.3.1. It contains fixes to the styles of annotated content.

##### connected issues and PRs:
[GN-5000](https://binnenland.atlassian.net/browse/GN-5000?atlOrigin=eyJpIjoiOWZlZjJmZjgxYmVjNDcxN2JhMTBmMjc0NWZkODRiZTgiLCJwIjoiaiJ9)
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/467

### How to test/reproduce
- Start the frontend
- Create a new agendapoint with a decision
- Notice that the annotated content of the decision (title, description etc.) has a full-width dotted bottom border.

### Challenges/uncertainties
Draft while still using dev version of plugins



### Checks PR readiness
